benchmarks/Dapper.Tests.Performance/Dapper.Tests.Performance.csproj

```diff
@@ -48,4 +48,12 @@
   <ItemGroup Condition="'$(TargetFramework)' != 'net462'">
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'net5.0'">
+    <PackageReference Include="Norm.net">
+      <Version>3.1.0</Version>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Benchmarks.Norm.cs" Condition="'$(TargetFramework)' != 'netcoreapp3.1' And '$(TargetFramework)' != 'net5.0'" />
+  </ItemGroup>
```

Comment:
Instead of all these one-off changes, please use the `<ItemGroup Condition="'$(TargetFramework)' != 'net462'">` section above for a normal include, then use `#if !NET4X` in the file itself. This will make it behave like other benchmarks overall.